iorgen/parser_rust.py

```diff
@@ -17,15 +18,19 @@
     "yield"
 ]
 
+# KEYWORDS extended with the list of identifiers used by the generated code
+EXTENDED_KEYWORDS = KEYWORDS + [
+    "buffer", "read_line", "std", "Box", "String", "Vec"
```

Comment:
Shouldn’t you add `Result` to the list? I’ve seen it used a lot and I fear for collision with a user struct.

Reply:
Nice catch!